build/config/brave_build.gni

```diff
@@ -4,3 +4,4 @@
 import("//brave/brave_repack_locales.gni")
 import("//brave/build/features.gni")
 import("//brave/net/sources.gni")
+import("//brave/third_party/blink/renderer/includes.gni")
```

Comment:
maybe `modules.gni` ?

Reply:
considered that- but it also includes `core` changes

Reply:
but its purpose is to  make modules work. If nothing else `config.gni` would be  a better choice, but it can be a follow-up